index.js

```diff
@@ -14,7 +16,7 @@ app.get('/', async (req, res) => {
 });
 
 // TODO change to post when finish testing
-app.get('/create-pdf', async (req, res) => {
+app.get('/certificate', async (req, res) => {
```

Comment:
Add optional query params - firstName lastName
and add logic as per 3. requirement in Telegram